content/en/blog/2023/announcing-notation-rc4.md

```diff
@@ -0,0 +1,129 @@
+
+```shell
+notation policy import my_policy.json
+```
+
+The `notation policy show` command allows users to easily view trust policy configuration and redirect the output to a file for sharing, updating, or other purposes. For example:
+
+```shell
+notation policy show
+```
+
+We will be introducing more commands in future releases. Stay tuned for updates on how we're making it easier for you to manage your trust policy configuration.
+
+### Support OCI distribution 1.1 referrers API
+
+The Referrers API is a new feature added in [OCI distribution spec v1.1-rc.1](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#listing-referrers), which allows clients to fetch a list of referrers in an efficient and clean manner. In the context of Notation, referrers are signatures that refer to the container image. Since this release, Notation verifies whether the Referrers API is available in the registry when pushing signatures into the registry. If the Referrers API is not available, Notation follows the [fallback procedure](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#unavailable-referrers-api) and updates the image index pushed to a tag described by the referrers tag schema.
+
+### Introduce signing and verification with OCI image layout as experimental feature
+
+Typically, images are pushed to registries before they are signed. However, if the registries are compromised, the images you signed could already be tampered with. These images could pass signature verification and be deployed in the production environment. To address this issue, we have introduced an experimental feature that allows users to sign images before pushing them to registries. This is especially valuable if the registries are not within your trust boundaries. The OCI image layout is a standard defined in the [OCI image spec 1.0](https://github.com/opencontainers/image-spec/blob/v1.0/image-layout.md). It is essentially a directory structure that contains files and folders that refer to an OCI image. Here's a glimpse of the experience on Linux, and we will release a tutorial for this feature soon.
+
+```shell
+# Enable experimental feature
+export NOTATION_EXPERIMENTAL=1
+# Create oci image layout as tarball file
+docker buildx build . -f Dockerfile -o type=oci,dest=hello-world.tar -t hello-world:v1
+# Extract tarball to a directory named hello-world, 
+# so that the signatures can be associated with the image
+tar -xf ./hello-world.tar -C hello-world
+# Sign OCI image layout
+notation sign --oci-layout ./hello-world:v1
+# List signatures
+notation list --oci-layout ./hello-world:v1
```

Comment:
I notice that `notation verify --oci-layout` is not included here. Every new feature should be added in the announcement doc.